test/views/TwoEyeReviewSpec.scala

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import play.api.inject.Injector
+import play.api.i18n.{Messages, MessagesApi, Lang}
+import play.api.test.FakeRequest
+import play.twirl.api.Html
```

Comment:
IntelliJ is reporting some unused imports.

Reply:
will remove